packages/styles/scss/patterns/simplelongform/_simplelongform.scss

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin simplelongform { 
+  .#{$prefix}--simplelongform {
+    color: $text-01;
+    background: $ui-background;
+    &__container {
+      @include carbon--make-container;
+    }
+    &__row {
+      @include carbon--make-row;
+    }
+    &__col {
+      @include carbon--make-col-ready;
+    }
+    &__title {
+      margin-top: $layout-05;
+      @include type-style('expressive-heading-05');
+    }
+    &__content {
+      margin-top: $layout-03;
+      @include type-style('body-long-02');
+    }
+    &__divider {
+      border-bottom: 1px solid $ui-04;
+      margin-bottom: $layout-01;
+      margin-top: $layout-06;
+      margin-left: -$layout-01;
+      margin-right: -$layout-01;
+      &__col {
+        @include carbon--make-col-ready;
+      }
+    }
+    &__link {
+      margin-top: $layout-03;
+      margin-left: -$layout-01;
+      margin-right: -$layout-01;
+      max-width: carbon--mini-units(40);
+      .bx--link-with-icon {
```

Comment:
These are a bunch of overrides, we should probably think about possibly extending these components perhaps so we don't have to do this within the pattern.

cc: @sannminwin @oliviaflory @photodow @kennylam @annawen1 @andysherman2121 @larahanlon2